README.md

```diff
@@ -8,6 +8,20 @@ infrastructure.
 Setup
 -----
 
+### Setting up a GitHub PAT
+
+As a prerequisite for running all scripts in this repo, you will need a GitHub Personal Access
+Token with full access to your account. You can generate one
+[here](https://github.com/settings/tokens).
+
+While creating the token, make sure at the least the scopes `repo`, and `workflow`.
```

Comment:
missing the end of the sentence, make it to this: `... make sure at the least the scopes `repo` and `workflow` are selected.`